src/api/.keystone/config.js

```diff
@@ -33,9 +33,33 @@ __export(keystone_exports, {
   default: () => keystone_default
 });
 module.exports = __toCommonJS(keystone_exports);
-var import_config2 = require("dotenv/config");
+var import_config3 = require("dotenv/config");
 var import_core2 = require("@keystone-6/core");
 
+// middleware/headers/check-api-key/index.ts
+var import_config = require("dotenv/config");
+var { API_KEY } = process.env;
+var checkApiKey = (req, res, next) => {
```

Comment:
I'm not familiar with keystone - why does this need tobe defined here and then also defined again i the check-api-key/index.ts file?

Reply:
Same as above, this is automatically compiled by KeystoneJS.

We have to include this in the repo unfortunately, we tried generating it behind the scenes with docker/GHA (to avoid having to commit this file), but we couldn't get it to work.